crates/turborepo-ui/src/color_selector.rs

```diff
@@ -1,21 +1,20 @@
 use std::{
     collections::HashMap,
+    hash::{DefaultHasher, Hash, Hasher},
     sync::{Arc, OnceLock, RwLock},
+    u8,
 };
 
 use console::{Style, StyledObject};
 
-static COLORS: OnceLock<[Style; 5]> = OnceLock::new();
+static COLORS: OnceLock<[Style; u8::MAX as usize]> = OnceLock::new();
 
-pub fn get_terminal_package_colors() -> &'static [Style; 5] {
+pub fn get_terminal_package_colors() -> &'static [Style; u8::MAX as usize] {
     COLORS.get_or_init(|| {
-        [
-            Style::new().cyan(),
-            Style::new().magenta(),
-            Style::new().green(),
-            Style::new().yellow(),
-            Style::new().blue(),
-        ]
+        let colors: [Style; u8::MAX as usize] =
+            core::array::from_fn(|index| Style::new().color256(index as u8));
+
+        colors
```

Comment:
I don't believe we want to use all of the available colors as:
 - many of them aren't especially nice to look at
 - many of the colors are very similar e.g. in my tests I got 4 very similar shades of red

<img width="384" alt="Screenshot 2024-08-14 at 1 32 23 PM" src="https://github.com/user-attachments/assets/604076b6-278d-4d48-8202-fc25172a4f9c">

Reply:
I'm still opposed to using all of the colors here for the reasons listed above. 

We can expand our list in a separate PR, but we need to make sure the colors selected are easy to read in most terminals.